src/Actions/Inform/Credits.cs

```diff
@@ -46,7 +47,7 @@ public override void Execute(ActionInput actionInput)
             {
                 if (cachedContents == null || (parameters.Length > 0 && parameters[0].ToLower() == "reload"))
                 {
-                    StreamReader reader = new StreamReader("Files\\Credits.txt");
+                    using StreamReader reader = new StreamReader(Path.Combine(GameConfiguration.DataStoragePath + "\\Credits.txt"));
```

Comment:
Path combine wants multiple args, it puts the correct environment path separator there instead of assuming it is backslash; more cross platform ready as 'DataStoragePath, "Credits.txt")'